core/iwasm/common/wasm_runtime_common.c

```diff
@@ -1183,7 +1183,7 @@ wasm_runtime_get_function_type(const WASMFunctionInstanceCommon *function,
 }
 
 WASMFunctionInstanceCommon *
-wasm_runtime_lookup_function(WASMModuleInstanceCommon *const module_inst,
+wasm_runtime_lookup_function(WASMModuleInstanceCommon *module_inst,
```

Comment:
Shall we not change the API's prototype? I had a test, it doesn't cause compile error or warning. Here using const is reasonable, and the API had been used for a long time.

Reply:
Could we not change the function prototype?